substrate/frame/revive/src/exec.rs

```diff
@@ -1373,6 +1373,13 @@ where
 	}
 }
 
+/// Determine if the given address is a precompile.
+/// For now, we consider that all addresses between 0x1 and 0xff are reserved for precompiles.
+fn is_precompile(address: &H160) -> bool {
+	let bytes = address.as_bytes();
+	bytes.starts_with(&[0u8; 19]) && (1..=255).contains(&bytes[19])
```

Comment:
Isn't this just a weird way of expressing this?

```suggestion
	bytes.starts_with(&[0u8; 19]) && bytes[19] != 0
```